kubernetes-deployment/test/test_maintainlib_common.py

```diff
@@ -119,6 +119,40 @@ def test_package(self):
 
 
 
+    def test_ipv4_address_validation_correct(self):
+
+        addr = "128.0.0.1"
+        self.assertTrue(common.ipv4_address_validation(addr))
+
+
+
+    def test_ipv4_address_validation_wrong(self):
+
+        addr1 = "128.0.0.x"
+        self.assertFalse(common.ipv4_address_validation(addr1))
+
+        addr2 = "256.0.0.0"
+        self.assertFalse(common.ipv4_address_validation(addr2))
+
+
+
+    def test_port_validation_correct(self):
+
+        port1 = 22
+        self.assertTrue(common.port_validation(port1))
+
+        port2 = "232"
+        self.assertTrue(common.port_validation(port2))
+
+
+
+    def test_port_validation_wrong(self):
```

Comment:
>test_port_validation_wrong [](start = 8, length = 26)

no need to use two tests for valid and invalid parameters. please merge the two into one.

Reply:
Done